docker/setup.sh

```diff
@@ -0,0 +1,3 @@
+#!/usr/bin/env bash
+
+git clone --branch develop-es7 https://github.com/hysds/hysds_commons.git
```

Comment:
@DustinKLo: update this since `develop` will be ES7 and `develop-es7` will go away.